testing/tests/DevExpress.ui.widgets.treeList/selection.tests.js

```diff
@@ -8,6 +10,8 @@ QUnit.testStart(function() {
 ';
 
     $('#qunit-fixture').html(markup);
+
+    addShadowDomStyles($('#qunit-fixture'));
```

Comment:
Why do we need additionally attach shadow dom styles? I thought it's made on component level

Reply:
We need to discuss this moment. Here and in some other places we use not components themselves, but this thing:
`this.rowsView.render($testElement);`
so, the `render` function is not called. Thats my I added styles manually, because I thought that the styles should be added only inside the `render` function of real component